api/health/index.ts

```diff
@@ -37,39 +34,28 @@ export const healthChecks = {
   },
 }
 
-/*if (showFeature(FEATURE_TERMS_AND_CONDITIONS_ENABLED)) {
-  healthChecks.checks = {...healthChecks.checks, ...{
-  termsAndConditions: checkServiceHealth(getConfigValue(SERVICES_TERMS_AND_CONDITIONS_URL)),
-    },
-  }
-}*/
-
-/**
- * Add Reform standard Health Checks for /health and /health/liveness
- * to the application.
- *
- * Note that these checks are hit in the Build pipelines.
- *
- * @param app
- */
 export const addReformHealthCheck = app => {
-
+  if (showFeature(FEATURE_TERMS_AND_CONDITIONS_ENABLED)) {
+    config.checks = {
+      ...config.checks, ...{
+        termsAndConditions: checkServiceHealth(getConfigValue(SERVICES_TERMS_AND_CONDITIONS_URL)),
+      },
+    }
+  }
   if (showFeature(FEATURE_REDIS_ENABLED)) {
     xuiNode.on(SESSION.EVENT.REDIS_CLIENT_READY, (redisClient: any) => {
-      app.locals.redisClient = redisClient
-      healthChecks.checks = {...healthChecks.checks, ...{
-          redis: healthcheck.raw(async () => {
-            const status = await redisHealth()
-            return status ? healthcheck.up() : healthcheck.down()
+      console.log('REDIS EVENT FIRED!!')
```

Comment:
I don't think we want this `console.log()` in production code.

Reply:
changed this to use the correct logger. trying to keep the logging in the same places as it does on the MO project.